backend/src/main/java/com/senior/project/backend/Activity/EventHandler.java

```diff
@@ -19,4 +19,10 @@ public EventHandler(EventService eventService){
     public Mono<ServerResponse> all(ServerRequest serverRequest) {
         return ServerResponse.ok().body(this.eventService.all(), Event.class );
     }
+
+    public Mono<ServerResponse> dashboard(ServerRequest serverRequest) {
+        serverRequest.queryParam("studentID");
```

Comment:
again we should get the student id from the auth token. We can handle that later